src/Workspaces/Remote/ServiceHub/Services/AsynchronousOperationListener/RemoteAsynchronousOperationListenerService.cs

```diff
@@ -11,8 +11,14 @@
 
 namespace Microsoft.CodeAnalysis.Remote
 {
-    internal class RemoteAsynchronousOperationListenerService : BrokeredServiceBase, IRemoteAsynchronousOperationListenerService
+    internal sealed class RemoteAsynchronousOperationListenerService : BrokeredServiceBase, IRemoteAsynchronousOperationListenerService
     {
+        internal sealed class Factory : FactoryBase<IRemoteAsynchronousOperationListenerService>
```

Comment:
📝 Previous location was correct. Nested types at the bottom or in a separate file.

Reply:
It was not correct. This is correct.